src/run/index.ts

```diff
@@ -133,6 +133,8 @@ export function executeBicepBuild(files: string[], bicepTool: string, additional
 
         if (bicepProcess.code !== 0) {
             throw new Error(`Failed to execute script. Related file: ${file}`);
+        } else if (bicepProcess.stderr.length > 0) {
+            taskLib.setResult(taskLib.TaskResult.SucceededWithIssues, bicepProcess.stderr, true);
```

Comment:
@svaenn could you change the `done` parameter and no provide it? I think it is better to continue with the compilation of all files and then report the result with warnings.

Now with that parameter set as `true`, the task will stop when a file has one or more warnings.

Reply:
I think is as simple as:

`taskLib.setResult(taskLib.TaskResult.SucceededWithIssues, bicepProcess.stderr);`

Reply:
@piraces Good point! Just pushed the update.